README.md

```diff
@@ -45,7 +54,71 @@ const document = {
   language: 'fr',
 };
 
-botmeter.indexDocument(document, function(error, body) {
+genericLogger.indexDocument(document, function(error, body) {
+  // Body will equal 'ok' if logging is successful
   ...
 });
 ```
+
+### With Messenger
+
+```node.js
+const BotmeterLogger = require('botmeter-logger');
+const request = require('request');
+
+const messengerLogger = new BotmeterLogger({
+  appId: 'id',
+  appKey: 'key',
+}).messenger;
+
+const response = {
```

Comment:
response is strange here, isn't it ? I would have expected query or something similar

Reply:
Yes, I agree, but to be honest I don’t clearly understand this code

Reply:
Oh I understand now, so this is the response of a messenger bot, I guess we don't know what's in the responseJson? It would be nice to try it ... No idea how long it takes. Anyway we can specify this by renaming the variable or commenting, overall it feels weird that you define the bot response (with a const) but the interesting object (responseJson) is unkown and undefined. Maybe we should put this above the sdk code, maybe apart from it, like : 
![image](https://user-images.githubusercontent.com/3613657/30057524-5091a804-9237-11e7-8e55-77c736caecbd.png)